airflow-core/src/airflow/dag_processing/collection.py

```diff
@@ -371,7 +374,10 @@ def update_dag_parsing_results_in_db(
     warnings: set[DagWarning],
     session: Session,
     *,
-    warning_types: tuple[DagWarningType] = (DagWarningType.NONEXISTENT_POOL,),
+    warning_types: tuple[DagWarningType, DagWarningType] = (
```

Comment:
```suggestion
    warning_types: tuple[DagWarningType, ...] = (
```
we should use this everywhere for warning_types so we can avoid signature change in the future